modules/about-base-training-images.adoc

```diff
@@ -0,0 +1,64 @@
+
+[role="_abstract"]
+The base training images for distributed workloads are optimized with the tools and libraries that you need to run distributed training jobs. 
+You can use the provided base images, or an {productname-short} admin user can create custom images adapted to your needs.
+
+ifndef::upstream[]
+For information about Red Hat support of training images and packages, see link:https://access.redhat.com/articles/rhoai-supported-configs[{productname-long}: Supported Configurations].
+endif::[]
+
+The following table lists the training images that are installed with {productname-long} by default.
+
+If the preinstalled packages that are provided in these images are not sufficient for your use case, you have the following options:
+
+* Install additional libraries after launching a default image. 
+This option is good if you want to add libraries on an ad·hoc basis as you run training jobs. 
+However, it can be challenging to manage the dependencies of installed libraries, and your changes are not saved when the workbench restarts.
+
+
+ifdef::upstream[]
+* Create a custom image that includes the additional libraries or packages. 
+For more information, see
+link:{odhdocshome}/working-with-distributed-workloads/#creating-a-custom-training-image_distributed-workloads[Creating a custom training image].
+endif::[]
+
+ifndef::upstream[]
+* Create a custom image that includes the additional libraries or packages. 
+For more information, see link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/creating-a-custom-training-image_distributed-workloads[Creating a custom training image].
+
+[IMPORTANT]
+====
+The ROCm-compatible Ray cluster training image is Developer Preview software only. 
+Developer Preview software is not supported by {org-name} in any way and is not functionally complete or production-ready. 
+Do not use Developer Preview software for production or business-critical workloads. 
+Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a {org-name} product offering. 
+Customers can use this software to test functionality and provide feedback during the development process. 
+This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. 
+{org-name} might provide ways to submit feedback on Developer Preview software without an associated SLA.
+For more information about the support scope of {org-name} Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+endif::[]
+
+.Default distributed workload base images
+[cols="1,6"]
+|===
+| Image type | Description
+
+| Ray CUDA
+| If you are working with compute-intensive data science models that require NVIDIA GPU support, you can use the Ray Compute Unified Device Architecture (CUDA) base image to gain access to the NVIDIA CUDA Toolkit. 
```

Comment:
I'd phrase this in a way that the choice is driven by the users, like "and want to accelerate it with NVIDIA GPU", as it's not a strict technical requirement.

Reply:
Updated as follows:
From:    "that require <accelerator> support" 
To:         "and you want to accelerate the model performance with <accelerators>"

Please advise if the new text should be rephrased.

Reply:
I like "If you are working with compute-intensive models and you want to accelerate the training job with <accelerator> support" as you had suggested async.